specification/azsadmin/resource-manager/subscriptions/readme.md

```diff
@@ -107,7 +107,7 @@ AutoRest V3 generators require the use of `--tag=all-api-versions` to select api
 
 This block is updated by an automatic script. Edits may be lost!
 
-``` yaml
+``` yaml $(tag) == 'all-api-versions
```

Comment:
single quote missing

Reply:
I added that quote, but forgot to push the change. I have updated it now.